samples/display/lvgl/boards/stm32f429i_disc1.conf

```diff
@@ -0,0 +1,10 @@
+# enable spi
+CONFIG_SPI=y
+
+# enable display
+CONFIG_ILI9340=y
+
+CONFIG_LVGL=y
```

Comment:
`CONFIG_LVGL` is already enabled by the sample prj.conf, https://github.com/zephyrproject-rtos/zephyr/blob/master/samples/display/lvgl/prj.conf